<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_login">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>211</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qradialgradient(cx:0.5, cy:0.5, radius:1.5, fx:0.499586, fy:0.5, stop:0 rgba(31, 217, 96, 255), stop:1 rgba(255, 255, 255, 255))
</string>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>148</red>
        <green>33</green>
        <blue>146</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>147</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>107</y>
       <width>125</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="loginTitle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>189</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;
  &lt;head&gt;
    &lt;style&gt;
      body {
        font-family: Arial, sans-serif;
        background-color: #f2f2f2;
      }
  p {
    color: #f8f8ff;
    font-size: 16px;
    line-height: 1.5;
  }
&lt;/style&gt;
  &lt;/head&gt;
  &lt;body&gt;
    &lt;p&gt;Welcome to SoundScape!&lt;/p&gt;
  &lt;/body&gt;
&lt;/html&gt;




</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
color:green;
border-style: outset;
border-radius: 10px;
border-width: 2px;
border-color: white;
font: QFont f(&quot;Helvetica&quot;);</string>
     </property>
     <property name="text">
      <string>Login!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>106</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>138</y>
       <width>125</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_playlists">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>169</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>My playlists:</string>
    </property>
    <widget class="QListView" name="listView_playlists">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>20</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
color: black;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_pushButtons">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>-10</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_playlists">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>Get My Playlists</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_createLocation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Create Location Zone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_spotify">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Connect to Spotify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refreshLocation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Refresh Location</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_createPMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Create Playlist Map</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_playlistMaps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>My Playlist Maps:</string>
     </property>
     <widget class="QListView" name="listView_playlistMaps">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>161</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;
color: black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_editLocation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Delete Location Zone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refreshLists">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>147</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(30,215,96);
color:black;
border-style: outset;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Refresh Lists</string>
     </property>
    </widget>
   </widget>
   <widget class="QQuickWidget" name="quickWidget_map">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>431</width>
      <height>381</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_locations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>169</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>My locations:</string>
    </property>
    <widget class="QListView" name="listView_locations">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
color: black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Location Manually</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
